src/crypto/store/indexeddb-crypto-store-backend.js

```diff
@@ -314,7 +314,10 @@ export class Backend {
         getReq.onsuccess = function() {
             const cursor = getReq.result;
             if (cursor) {
-                results[cursor.value.sessionId] = cursor.value.session;
+                results[cursor.value.sessionId] = {
+                    session: cursor.value.session,
+                    lastReceivedMessagets: cursor.value.lastReceivedMessageTs,
```

Comment:
Why is the case changed here (lastReceivedMessage*t*s vs lastReceivedmessage*T*s)?

Reply:
ah, good spot